xio-core/pom.xml

```diff
@@ -464,6 +464,13 @@
         <groupId>org.eluder.coveralls</groupId>
         <artifactId>coveralls-maven-plugin</artifactId>
         <version>4.3.0</version>
+        <dependencies>
+          <dependency>
+            <groupId>javax.xml.bind</groupId>
```

Comment:
is this for java9?

Reply:
yes they took this out of the JVM, it's how a lot of projects were doing base64 without pulling in another jar.